bundles/mod-mda-Degrade.lv2/Degrade.ttl

```diff
@@ -30,116 +30,143 @@ mda:Degrade
         lv2:index 0 ;
         lv2:name "Headroom" ;
         lv2:symbol "headroom" ;
-        lv2:default 0.8 ;
-        lv2:minimum 0.0 ;
-        lv2:maximum 1.0 ;
-#        units:unit units:db ;
+        lv2:default -6.0 ;
+        lv2:minimum -30.0 ;
+        lv2:maximum 0.0 ;
+        units:unit units:db ;
         rdfs:comment "Peak clipping threshold"
     ] , [
         a lv2:InputPort ,
             lv2:ControlPort ;
         lv2:index 1 ;
         lv2:name "Quant" ;
         lv2:symbol "quant" ;
-#        lv2:portProperty lv2:integer ;
-        lv2:default 0.5 ;
-        lv2:minimum 0.0 ;
-        lv2:maximum 1.0 ;
-#        units:unit [
-#        rdfs:label   "bits" ;
-#        units:symbol "bit" ;
-#        units:render "%f bits"
-#        ] ;
+        lv2:portProperty lv2:integer ;
+        lv2:default 10 ;
+        lv2:minimum 4 ;
+        lv2:maximum 16 ;
+        units:unit [
+          rdfs:label   "bits" ;
+          units:symbol "bit" ;
+          units:render "%f bits"
+        ] ;
         rdfs:comment """Bit depth (typically 8 or below for "telephone" quality)"""
     ] , [
         a lv2:InputPort ,
             lv2:ControlPort ;
         lv2:index 2 ;
         lv2:name "Rate" ;
         lv2:symbol "rate" ;
-#        lv2:portProperty lv2:integer ;
-#        lv2:portProperty epp:logarithmic ;
-        lv2:default 0.65 ;
-        lv2:minimum 0.0 ;
-        lv2:maximum 1.0 ;
+        lv2:portProperty lv2:integer ;
+        lv2:portProperty epp:logarithmic ;
+        lv2:default 48000 ;
+        lv2:minimum 4800 ;
+        lv2:maximum 48000 ;
         units:unit [
-        rdfs:label   "ssh" ;
-        units:symbol "s_sh" ;
-        units:render "%f S<>S&&H"
+          rdfs:label   "ssh" ;
+          units:symbol "s_sh" ;
+          units:render "%f Hz"
         ] ;
         rdfs:comment "Sample Rate (turn control to left or right for different sound characters)"
     ] , [
         a lv2:InputPort ,
             lv2:ControlPort ;
         lv2:index 3 ;
+        lv2:name "Integrator" ;
+        lv2:symbol "integrator_sr" ;
+        lv2:default 0 ;
+        lv2:minimum 0 ;
+        lv2:maximum 1 ;
+        lv2:portProperty lv2:enumeration , lv2:integer ;
+        lv2:scalePoint [
+            rdfs:label "Integrator Off" ;
+            rdf:value 0
+        ] , [
+            rdfs:label "Integrator On" ;
+            rdf:value 1
+        ]
+    ] , [
+        a lv2:InputPort ,
+            lv2:ControlPort ;
+        lv2:index 4 ;
         lv2:name "Post Filter" ;
         lv2:symbol "post_filt" ;
-#        lv2:portProperty lv2:integer ;
-#        lv2:portProperty epp:logarithmic ;
-        lv2:default 0.9 ;
-        lv2:minimum 0.0 ;
-        lv2:maximum 1.0 ;
-#        units:unit units:db ;
+        lv2:portProperty lv2:integer ;
+        lv2:portProperty epp:logarithmic ;
+        lv2:default 15000 ;
+        lv2:minimum 200 ;
+        lv2:maximum 20000 ;
+        units:unit units:Hz ;
```

Comment:
`units:Hz` is not a valid property, use `units:hz` instead.
See http://lv2plug.in/ns/extensions/units#hz